### DISCUSSION TITLE
Unable to submit bug report in github - Disabled ipv6 breaks netbox

### DISCUSSION THREAD

**Discussion by oceania-npc** [netbox-community/netbox#11691] [Help Wanted!]
Trying to submit a bug report in github.  Filled out every field in the form but the submit button at the bottom is greyed out (tried three different browsers).

Bug:

Title:  Netbox does not load when ipv6 disabled on Ubuntu 20.04

Netbox version:
3.4.4

Python version: 
3.8

Steps to Reproduce:
On Ubuntu 20.04, if ipv6 is disabled in grub, netbox does not load.
1. Disable ipv6 in grub
    `vim /etc/default/grub
     GRUB_CMDLINE_LINUX_DEFAULT="ipv6.disable=1" # use spaces to separate items if needed`
2. Update grub
`sudo update-grub`
3.  Reboot
4. Attempt to open netbox in browser.

Expected Behavior:
Netbox should load if ipv6 is disabled.

Observed Behavior:
Netbox fails to load after ipv6 disabled in grub.  Re-enabling ipv6, netbox loads.

**Comment by kkthxbye-code** [comment]
This is the wrong repository, you want the netbox-docker one: https://github.com/netbox-community/netbox-docker

Also it is in the wiki: https://github.com/netbox-community/netbox-docker/wiki/Troubleshooting#netbox-container-doesnt-start-when-ipv6-is-disabled

**Reply by oceania-npc** [reply]
I'm not using netbox docker.  I'm using source downloaded from here:
https://github.com/netbox-community/netbox

**Reply by kkthxbye-code** [reply]
Sorry, I misunderstood. There's nothing preventing netbox running with IPv6 disabled, but you could try expanding on your discussion with information regarding how you are running netbox and what the logs say when it "breaks". It's very hard to guess the issue with so little information provided, you essentially just said "it doesn't work".

**Reply by jeremystretch** [reply]
The wiki article linked above still likely addresses your issue if using the provided nginx config. This is a configuration problem, not a NetBox bug.

**Reply by oceania-npc** [reply]
nginx.conf shows ipv6only=off.  Am I missing something?

contrib/nginx.conf
server {
    listen [::]:443 ssl ipv6only=off;

    # CHANGE THIS TO YOUR SERVER'S NAME
    server_name netbox.example.com;

    ssl_certificate /etc/ssl/certs/netbox.crt;
    ssl_certificate_key /etc/ssl/private/netbox.key;

    client_max_body_size 25m;

    location /static/ {
        alias /opt/netbox/netbox/static/;
    }

    location / {
        proxy_pass http://127.0.0.1:8001;
        proxy_set_header X-Forwarded-Host $http_host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-Proto $scheme;
    }
}

server {
    # Redirect HTTP traffic to HTTPS
    listen [::]:80 ipv6only=off;
    server_name _;
    return 301 https://$host$request_uri;
}

**Reply by markkuleinio** [reply]
`[::]` = IPv6 address

See for example https://serverfault.com/questions/638367/do-you-need-separate-ipv4-and-ipv6-listen-directives-in-nginx for ideas how to change the configuration (`listen 80;`).
